presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -288,7 +288,7 @@ Property Name                                Description
                                              connect to an S3-compatible storage system instead
                                              of AWS. When using v4 signatures, it is recommended to
                                              set this to the AWS region-specific endpoint
-                                             (e.g., ``http[s]://<bucket>.s3-<AWS-region>.amazonaws.com``).
+                                             (e.g., ``http[s]://s3.<AWS-region>.amazonaws.com``).
```

Comment:
```suggestion
                                             (for example, ``http[s]://s3.<AWS-region>.amazonaws.com``).
```
Do not use Latin abbreviations. See [the Gitlab documentation style guide word list entry for e.g.](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#eg).